### DISCUSSION TITLE
Does Spiral Builder Contain All Pre-Configuration Features

### DISCUSSION THREAD

**Discussion by chaibronz** [SpiralLinux/SpiralLinux-project#373] [Q&A] [answered]
I'm interested in running a Debian install of Sway and it looks like Spiral Builder could be a good starting point for that. 

What I'm wondering is if the Builder version would still come with all the Spiral pre-configurations such as, btrfs subvolume layout w/ automated snapshots bootable via GRUB, pipewire with low latency config, improved font rendering, zram swap etc .. I understand Builder only comes with a basic icewm environment and no theming, which is fine, but wasn't sure if any thing else would be left out.

**Comment by geckolinux** [comment, marked as answer]
Hi there, yes it does contain all of those features. Basically the Builder edition includes all the low-level stuff that can't be achieved via GUI settings in the desktop environment. Regarding font rendering, it does contain some tweaks in `/etc/fonts/conf.d/` , but you would have to choose the font style and rendering choices in Sway (I recommend Ubuntu with "slight" hinting and Rgba antialiasing).

**Reply by chaibronz** [reply]
Perfect, thanks for confirming!
